unlock-app/src/components/interface/LatestBlogs.tsx

```diff
@@ -0,0 +1,137 @@
+import axios from 'axios'
+import dayjs from 'dayjs'
+import isToday from 'dayjs/plugin/isToday'
+dayjs.extend(isToday)
+
+export interface Blog {
+  title: string
+  link: string
+  id: string
+  updated: string
+  viewed: boolean
+}
+
+export function BlogLink({
+  setBlogs,
+  blog,
+}: {
+  setBlogs: (blogs: Blog[]) => void
+  blog: Blog
+}) {
+  return (
+    <a
+      href={blog.link}
+      target="_blank"
+      onClick={() => updateBlog(blog.id, setBlogs)}
+      rel="noopener noreferrer"
+      className="block"
+    >
+      <div className="flex flex-col gap-1">
+        <div
+          className={`font-medium ${blog.viewed ? 'text-gray-400' : 'text-black'}`}
+        >
+          {blog.title}
+        </div>
+        <div className="text-sm text-gray-500">
+          {dayjs(blog.updated).format('MMM DD, YYYY')}
+        </div>
+      </div>
+    </a>
+  )
+}
+
+export function getBlogs() {
+  if (typeof localStorage !== 'undefined') {
+    const data =
+      localStorage.getItem('latest_blogs') &&
+      JSON.parse(localStorage.getItem('latest_blogs')!)
+    return data
+  }
```

Comment:
Please don't use that. We have a `useLocalStorage` hook. Use our exsting library please!